packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -214,12 +300,18 @@ const GeneratedDesignButtonContainer: React.FC< GeneratedDesignButtonContainerPr
 		use_screenshot_overrides: true,
 	} );
 
+	const divTrackingRef = useTrackDesignView( {
```

Comment:
Why do we call it `divTrackingRef` here instead of `trackingDivRef`? Could you make it consistent? Also, as it's a callback ref, it might be better to have something like `set` prefix, e.g: `setTrackingDivRef`

Reply:
This was an oversight on my part. I've updated it to `trackingDivRef` to be consistent.

> Also, as it's a callback ref, it might be better to have something like `set` prefix, e.g: `setTrackingDivRef`

I thought about the naming, and left it as the more basic/general `*Ref` naming because the fact that it's a callback ref is mostly an internal implementation detail of `useTrackDesignView()`, and using a `set` prefix would suggest that this is a state-related callback. We could update this to something like `trackingDivCallbackRef`, but I don't think that buys us much in terms of clarity.